zjit/src/backend/arm64/mod.rs

```diff
@@ -835,10 +835,8 @@ impl Assembler {
         }
 
         /// Emit a CBZ or CBNZ which branches when a register is zero or non-zero
-        fn emit_cmp_zero_jump(_cb: &mut CodeBlock, _reg: A64Opnd, _branch_if_zero: bool, target: Target) {
-            if let Target::Label(_) = target {
-                unimplemented!("this should be re-implemented with Label for side exits");
-                /*
+        fn emit_cmp_zero_jump(cb: &mut CodeBlock, reg: A64Opnd, branch_if_zero: bool, target: Target) {
+            if let Target::CodePtr(dst_ptr) = target {
```

Comment:
Not directly related to this change: does it make sense to have counter tracking how much the unimplemented backend codepath is hit to help prioritizing?